app/src/test/java/org/simple/clinic/summary/bloodsugar/BloodSugarSummaryViewEffectHandlerTest.kt

```diff
@@ -27,4 +30,14 @@ class BloodSugarSummaryViewEffectHandlerTest {
     //then
     testCase.assertOutgoingEvents(BloodSugarSummaryFetched(measurements))
```

Comment:
Can we add a `verifyZeroInteractions(ui)` here?

Reply:
Done